core/storage/question/gae_models.py

```diff
@@ -103,3 +103,45 @@ def create(
             language_code=language_code)
 
         return question_model_instance
+
+
+class QuestionSkillLinkModel(base_models.VersionedModel):
+    """Model for storing Question-Skill Links.
+
+    The ID of instances of this class has the form
+    {{random_hash_of_16_chars}}
```

Comment:
Isn't this 12 by default?

Reply:
So I looked at the  _get_new_id function which says its a 16 character hash, thats why I wrote this comment.

Reply:
Ah, you mean for QuestionModel? I took at look and it refers to base_models.ID_LENGTH which is 12, so I think that comment is wrong. If you could change it, that'd be great!

(Note that you can inspect the local datastore at localhost:8000 to make sure.)

Reply:
Okay thanks for letting me know, changed the comment

Reply:
Comment is still wrong; did you push your changes?

Reply:
My changes should be pushed?

Reply:
Yep! Looks good now, thanks.